quantum/process_keycode/process_unicode_common.c

```diff
@@ -148,55 +127,60 @@ void send_unicode_hex_string(const char *str) {
   }
 }
 
-bool process_record_unicode_common(uint16_t keycode, keyrecord_t *record) {
+bool process_unicode_common(uint16_t keycode, keyrecord_t *record) {
   if (record->event.pressed) {
     switch (keycode) {
-      case UNICODE_MODE_OSX:
+    case UNICODE_MODE_OSX:
       set_unicode_input_mode(UC_OSX);
-        #if defined(AUDIO_ENABLE) && defined(UNICODE_SONG_OSX)
-          PLAY_SONG(osx_song);
-        #endif
-        break;
-      case UNICODE_MODE_LNX:
-        set_unicode_input_mode(UC_LNX);
-        #if defined(AUDIO_ENABLE) && defined(UNICODE_SONG_LNX)
-          PLAY_SONG(song_lnx);
-        #endif
-        break;
-      case UNICODE_MODE_BSD:
-        set_unicode_input_mode(UC_BSD);
-        #if defined(AUDIO_ENABLE) && defined(UNICODE_SONG_BSD)
-          PLAY_SONG(song_bsd);
-        #endif
-        break;
-      case UNICODE_MODE_WIN:
-        set_unicode_input_mode(UC_WIN);
-        #if defined(AUDIO_ENABLE) && defined(UNICODE_SONG_WINDOWS)
-          PLAY_SONG(windows_song);
-        #endif
-        break;
-      case UNICODE_MODE_WINC:
-        set_unicode_input_mode(UC_WINC);
-        #if defined(AUDIO_ENABLE) && defined(UNICODE_SONG_WIN_COMPOSE)
-          PLAY_SONG(win_compose_song);
-        #endif
-        break;
-      case UNICODE_MODE_OSX_RALT:
-        set_unicode_input_mode(UC_OSX_RALT);
-        #if defined(AUDIO_ENABLE) && defined(UNICODE_SONG_OSX_RALT)
-          PLAY_SONG(osx_ralt_song);
-        #endif
-        break;
+#if defined(AUDIO_ENABLE) && defined(UNICODE_SONG_OSX)
+      static float song_osx[][2] = UNICODE_SONG_OSX;
+      PLAY_SONG(song_osx);
+#endif
```

Comment:
The reason why I prefer preprocessor ifs (and preprocessor directives in general) to have their own separate indentation levels is that they aren't part of the program logic — they just pick & choose which parts of the code get compiled and which don't. Therefore, they shouldn't affect the indentation of the code itself.